src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Calculator.css';
+import PropTypes from 'prop-types';
+
+// Child component
+function Button({ label }) {
+  return <button type="button" className="operators">{label}</button>;
+}
+
+Button.propTypes = {
+  label: PropTypes.string.isRequired,
+};
+
+// Calculator component
+function Calculator() {
+  return (
+    <section className="calc-container">
+      <div className="display">0</div>
+      <div className="keyboard">
+
+        <div className="numbers">
```

Comment:
To make the UI of you calculator match the given design, consider removing the black borders around the buttons

The template
![Screenshot](https://github.com/JuanDBta/math-magicians/assets/84930355/d4d6ab8e-84b3-468f-b19a-dbaec7a57e68)

Your design
![Screenshot (1)](https://github.com/JuanDBta/math-magicians/assets/84930355/22796083-ef0b-4a47-9c80-a193e7a73666)